src/extrinsic/mod.rs

```diff
@@ -88,12 +91,11 @@ macro_rules! compose_extrinsic_offline {
 
         let signature = raw_payload.using_encoded(|payload| $signer.sign(payload));
 
-        let mut arr: [u8; 32] = Default::default();
-        arr.clone_from_slice($signer.public().as_ref());
+        let multi_signer: MultiSigner = $signer.public().into();
```

Comment:
Very nice, I hated this `arr.clone_from_slice`

Reply:
Allows for ECDSA signing too!